### PR TITLE
Localized & AppUiSettings refactoring

### PR DESCRIPTION
**Pull request by Minyewoo**

### CODE REVIEW

File: lib/src/core/string_loader.dart [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import 'dart:io';
+import 'package:flutter/services.dart';
+///
+abstract class StringLoader {
```
Comment (a-givertzman) [inline lib/src/core/string_loader.dart:4]: how about rename to JsonFile or JsonData

File: lib/src/core/string_loader.dart [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  const factory StringLoader.fromFile(String filePath) = _FileStringLoader;
+  const factory StringLoader.fromAsset(String assetPath) = _AssetStringLoader;
+  Future<String> load();
```
Comment (a-givertzman) [inline lib/src/core/string_loader.dart:7]: then method will be content or asString

File: lib/src/core/string_loader.dart (hunk 1 above)
Comment (Minyewoo) [reply lib/src/core/string_loader.dart]: Done

File: lib/src/core/string_loader.dart (hunk 2 above)
Comment (Minyewoo) [reply lib/src/core/string_loader.dart]: Done

**Review by a-givertzman — DISMISSED** [review]

**Pull request merged**